src/main/java/by/itech/lab/supplier/dto/mapper/RetailerMapper.java

```diff
@@ -0,0 +1,43 @@
+package by.itech.lab.supplier.dto.mapper;
+
+import by.itech.lab.supplier.domain.Retailer;
+import by.itech.lab.supplier.dto.RetailerDto;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+public class RetailerMapper implements BaseMapper<Retailer, RetailerDto> {
+
+    @Override
+    public RetailerDto map(Retailer retailer) {
+        return RetailerDto.builder()
+                .id(retailer.getId())
+                .fullName(retailer.getFullName())
+                .identifier(retailer.getIdentifier())
+                .deletedAt(retailer.getDeletedAt())
+                .active(retailer.getActive())
+                .customerId(retailer.getCustomerId())
+                .build();
+    }
+
+    public void update(final RetailerDto from, final Retailer to) {
```

Comment:
We could think of creating an interface for this mapper and override this method from the interface instead of direct usage public method from the mapper.